docs/tutorial/process-model.md

```diff
@@ -0,0 +1,203 @@
+# Process Model
+
+Electron inherits its multi-process architecture from Chromium, which makes the framework
+architecturally very similar to a modern web browser. In this guide, we'll expound on
+the conceptual knowledge of Electron that we applied in the minimal quick start app.
+
+## Why not a single process?
+
+Web browsers are incredibly complicated applications. Aside from their primary ability
+to display web content, they have many secondary responsibilities,
+such as managing multiple windows (or tabs) and loading third-party extensions.
+
+In the earlier days, browsers usually used a single process for all of this
+functionality. Although this pattern meant less overhead for each tab you had open,
+it also meant that one website crashing or hanging would affect the entire browser.
+
+## The multi-process model
+
+To solve this problem, the Chrome team decided that each tab would render in its own
+process, limiting the harm that buggy or malicious code on a web page could cause to
+the app as a whole. A single browser process then controls these processes, as well
+as the application lifecycle as a whole. This diagram below from the [Chrome Comic]
+visualizes this model:
+
+![Chrome's multi-process architecture](../images/chrome-processes.png)
+
+Electron applications are structured very similarly. As an app developer, you control
+two types of processes: main and renderer. These are analogous to Chrome's own browser
+and renderer processes outlined above.
+
+[Chrome Comic]: https://www.google.com/googlebooks/chrome/
+
+## The main process
+
+Each Electron app has a single main process, which acts as the application's entry
+point. The main process runs in a Node.js environment, meaning it has the ability
+to `require` modules and use all of Node.js APIs.
+
+### Window management
+
+The main process' primary purpose is to create and manage application windows with the
+[`BrowserWindow`][browser-window] module.
+
+Each instance of the `BrowserWindow` class creates an application window that loads
+a web page in a separate renderer process. You can interact with this web content
+from the main process using the window's [`webContents`][web-contents] object.
+
+> Note: A renderer process is also created for [web embeds][web-embed] such as the
+> `BrowserView` module. The `webContents` object is also accessible for embedded
+> web content.
+
+Because the `BrowserWindow` module is an [`EventEmitter`][event-emitter], you can also
+add handlers for various user events (for example, minimizing or maximizing your window).
+
+```js title='main.js'
+const { BrowserWindow } = require('electron')
+
+const win = new BrowserWindow({ width: 800, height: 1500 })
+win.loadURL('https://github.com')
+
+const contents = win.webContents
+console.log(contents)
```

Comment:
I'm not entirely sure what this has to do with the previous paragraph. Perhaps move the code block before the web embeds note?